ovos_plugin_manager/templates/audio.py

```diff
@@ -7,10 +7,42 @@
 
 from ovos_plugin_manager.templates.media import AudioPlayerBackend as _AB
```

Comment:
as mentioned in other comment, i suggest removing this import and reverting the base class to not subclass from this